src/wallet/config.rs

```diff
@@ -0,0 +1,166 @@
+use std::collections::HashMap;
+use std::error::Error;
+use std::fs::File;
+use std::io::{self, BufRead};
+
+use crate::{maker::config::MakerConfig, taker::TakerConfig};
+
+#[derive(Debug)]
+pub struct Config {
+    pub taker_config: TakerConfig,
+    pub maker_config: MakerConfig,
+}
+
+fn parse_toml(file_path: &str) -> io::Result<HashMap<String, HashMap<String, String>>> {
```

Comment:
This and `load_config` sounds like utlity. Doesn't fir in `Wallet` module.

Because the exact path ahead for this wallet is TBD, its better not to create a `wallet::config`.